File: app/src/main/java/com/tmontovaneli/picpaychallenge/service/UserService.java
package com.tmontovaneli.picpaychallenge.service;

import com.tmontovaneli.picpaychallenge.model.User;

import java.util.List;

import retrofit2.Call;
import retrofit2.http.GET;

/**
 * Created by tmontovaneli on 21/08/17.
 */

public interface UserService {


    @GET("users")
    Call<List<User>> listRepos();


}

File: app/src/main/java/com/v1pi/picpay_teste/Services/UserService.kt
package com.v1pi.picpay_teste.Services

import com.v1pi.picpay_teste.Domains.User
import io.reactivex.Flowable
import retrofit2.Call
import retrofit2.http.GET

interface UserService {
    @GET("users")
    fun users() : Flowable<List<User>>
}